doc/webservices/extract_api.rst

```diff
@@ -0,0 +1,478 @@
+2. You then have to regularly poll the :ref:`/iap/invoice_extract/get_results <webservices/extract_api/invoice_get_results>` route to get the document's parsing status.
+3. Once the result received, you can validate it by calling :ref:`/iap/invoice_extract/validate <webservices/extract_api/invoice_validate>` and sending the expected values. This step is optional but greatly helps the system to improve.
+
+These 3 routes are detailed in the next section. A python implementation of the full flow and a token for integration testing are provided at the end of the page.
+
+Routes
+------
+.. _webservices/extract_api/invoice_parse:
+
+/iap/invoice_extract/parse (POST)
+'''''''''''''''''''''''''''''''''
+
+Description
+^^^^^^^^^^^
+Request a processing of the document from the OCR. The route will return a `document_id` you can use to obtain the result of your request.
+
+Request Body
+^^^^^^^^^^^^
+
+* ``jsonrpc``: A string specifying the version of the JSON-RPC protocol. Must be exactly “2.0”. **REQUIRED**
+* ``method``: A string containing the name of the method to be invoked. Must be “call”. **REQUIRED**
+* ``params``:
+    * ``account_token``: token of the account from which credits will be taken. Each successful call costs one token. **REQUIRED**
+    * ``version``: The version will determine the format of your requests and the format of the server response. Some results can be unavailable in older versions. For the current version 1.2.0, send ‘120’. If not specified, the latest version will be used. **OPTIONAL**.
+    * ``documents``: The invoice must be provided as a string in the ASCII format. The list should contain only one string. If multiple strings are provided only the first string corresponding to a pdf will be processed. If no pdf is found, the first string will be processed. This field is a list only for legacy reasons. The supported extensions are *pdf*, *png*, *jpg* and *bmp*. **REQUIRED**
+    * ``user_infos``: 
+        Information concerning the person to whom the invoice is intended. This informations is not required in order for the service to work but it greatly improves the performance. **REQUIRED** (can be empty).
+        
+        - ``user_company_vat``: VAT number of the client. **OPTIONAL**
+        - ``user_company_name``: Name of the client’s company. **OPTIONAL**
+        - ``user_company_country_code``: Country code of the client. Format: `ISO3166 alpha-2 <https://www.iban.com/country-codes>`_. **OPTIONAL**
+        - ``user_lang``: client language. Format: *language_code + _ + locale* (ex: fr_FR, en_US). **OPTIONAL**
+        - ``user_email``: client email. **OPTIONAL**
+* ``id``: An identifier established by the client. It allows the client to keep track of which response goes with which request. This makes asynchronous calls easier. **REQUIRED**
+
+.. rst-class:: setup doc-aside
+
+.. switcher::
```

Comment:
Is this there on purpose? For the "JSON" title maybe?

Reply:
Yes for the JSON title. But I don't know if it is really necessary since all the payload displayed in this column are JSON